microsoft-edge/extensions/publish/publish-extension.md

```diff
@@ -185,48 +185,52 @@ Define the specific markets in which you plan to offer your extension.  The defa
+   | **Screenshots** | - | You can submit a maximum of 6 screenshots describing the functionality of the extension in detail.  The size of the screenshots must be either 640 x 480 pixels or 1280 x 800 pixels.  You can copy a screenshot from one language to all other languages by using the **Duplicate** option, which is displayed below the field after you upload at least one screenshot. |
+   | **YouTube video URL** | - | You can include a promotional YouTube video of the extension.  The URL and thumbnail of the YouTube video is displayed on the extension's store listing page. |
+   | **Short description** | Required for at least one language | To edit the short description, you must update the description field in the manifest file of the extension package, and then re-upload the package. |
+   | **Search terms** | - | Single words or phrases that help users discover the extension when searching at Microsoft Edge Add-ons.  The search terms aren't displayed to users. |
 
-   The information provided in the **Store Listings** page is displayed to users who view your listing at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com).  Even though some fields are optional, you should provide as much information as possible.
+   The information that's provided in the **Details for \<Language\>** page is displayed to users who view the extension's listing at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com).  Even though some fields are optional, you should provide as much information as possible.
 
-<!-- 
-1. Click **Save** in the upper right. todo: confirm UI
--->
+   The extension package contains the manifest file; to re-upload the manifest, you must re-upload the extension package.  All the other items (such as logo or tiles) are uploaded separately by using the **Details for \<Language\>** page.
+
+1. Click **Save draft** in the upper right.
 
 
 <!-- ------------------------------ -->
 #### Required properties
 
 To list your extension in the store, properties that are not indicated as Required in the above table are Optional.
 
-The following Required properties must be provided for _each_ language (such as U.S. English; `en-US`) that's listed in your extension zip package:
+The following Required properties must be provided for _each_ language (such as U.S. English; `en-US`) that's listed in the extension's zip package:
 * **Description**
-* **Extension Store logo**
+* **Extension logo**
 
-The following Required properties must be filled in for _at least one_ of the languages that's listed in your extension zip package:
-* **Display name**
+The following Required properties must be filled in for _at least one_ of the languages that's listed in the extension's zip package:
+* **Extension name**
```

Comment:
changed from **Display name** to **Extension name** like in table 5

fyi @G-u-r-j-e-e-t